CDM-Configuration.md

```diff
@@ -0,0 +1,114 @@
+```
+
+You will need to modify the URL above to point your instance of WebAPI running on `<server:port>` (default in this guide is `localhost:8080`) and then substitute the values specific to your CDM setup:
+
+- `<your_cdm_database_dialect>`: This is one of the following: `oracle`, `postgresql`, `pdw`, `redshift`, `impala`, `netezza`, `bigquery`, or `sql server` and is based on the platform you use to host your CDM.
+- `<your_vocab_schema>`: The schema containing your vocabulary tables
+- `<your_results_schema>`: The schema containing your results tables
+- `<your_temp_schema>`: The schema that it utilized for your temporary schema
+- The `initConceptHierarchy` value in the URL is set to `true` and is used to establish the concept_hierarchy which is a cached version of the OMOP vocabulary specific to the concepts found in your CDM. This table can take a while to build and only needs to be established one time. This value can be set to `false` if you do not need to re-establish this table. 
+
+Once you have created the URL for your environment, open a browser and navigate to that URL. The resulting SQL will be displayed in the browser and your database administrator can use this script to establish the results schema.
+
+## source and source_daimon table setup
+
+The WebAPI `source` and `source_daimon` tables were created when you started the tomcat service with the WebAPI war deployed.  These tables must be populated with a JDBC `source` connection and corresponding `source_daimons` that specify the location for the `cdm`, `vocabulary`, `results` and `temp` schemas associated to the source in order to use the OHDSI tools. For this example it is assumed that the CDM and Vocabulary exist as a separate schema in the same database instance.  
+
+### Example WebAPI SOURCE and SOURCE_DAIMON Inserts
+
+```sql
+INSERT INTO ohdsi.source (source_id, source_name, source_key, source_connection, source_dialect) VALUES (1, 'My Cdm', 'MY_CDM', ' jdbc:postgresql://server:5432/cdm?user={user}&password={password}', 'postgresql');
+
+INSERT INTO webapi.source_daimon (source_daimon_id, source_id, daimon_type, table_qualifier, priority) VALUES (1,1,0, 'cdm', 0);
+INSERT INTO webapi.source_daimon (source_daimon_id, source_id, daimon_type, table_qualifier, priority) VALUES (2,1,1, 'vocab', 0);
+INSERT INTO webapi.source_daimon (source_daimon_id, source_id, daimon_type, table_qualifier, priority) VALUES (3,1,2, 'results', 0);
+INSERT INTO webapi.source_daimon (source_daimon_id, source_id, daimon_type, table_qualifier, priority) VALUES (4,1,5, 'temp', 0);
+
+```
+
+The above inserts creates a source with `source_id = 1` with 4 daimon entries, one for each daimon type (0 = CDM, 1 = Vocabulary, 2 = Results, 5 = TEMP). If you'd like to configure more than 1 source for use in WebAPI, repeat the steps above and increment the `source_id` used to distinguish the sources from one another.
+
+👉 _Note: WebAPI will need to be restarted in order to see the the new sources._
```

Comment:
The endpoint: /WebAPI/sources/refresh will refresh the sources without WebAPI restart.

Reply:
Good catch! Revised per https://github.com/OHDSI/webapi-wiki/pull/1/commits/60a35b41d2765429611d87033b7c673e36c11dea